server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/PushSubscriptionSetCreatePerformer.scala

```diff
@@ -70,6 +75,9 @@ class PushSubscriptionSetCreatePerformer @Inject()(pushSubscriptionRepository: P
 
   private def create(clientId: PushSubscriptionCreationId, request: PushSubscriptionCreationRequest, mailboxSession: MailboxSession): SMono[CreationResult] =
     SMono.fromPublisher(pushSubscriptionRepository.save(mailboxSession.getUser, request))
+      .flatMap(subscription => verificationCreateProcessor.pushVerificationToPushServer(subscription.url,
```

Comment:
Should we handler the `pushVerificationToPushServer` error case? Then we will remove the entry saved before?